internal/voting/events_test.go

```diff
@@ -188,7 +179,7 @@ func Test_EventStore(t *testing.T) {
 			require.NoError(t, err)
 			defer cleanup()
 
-			err = voting.InitializeVoteStore(ctx, db)
+			err = InitializeVoteStore(ctx, db)
 			require.NoError(t, err)
 
 			e, err := NewEventStore(ctx, db) // needs BeginReadTx
```

Comment:
One of the main things I though this work could help improve was the burden of having the two constructors, and knowing that one needs to be initialized before the other.

Now that there are queries that JOIN between the schemas, it no longer makes sense to be have separate "init" or constructors.  It was already a problem before because event store methods required voting store to be initialized since it used query functions from `voting`, and now we're solidifying that dependency.

I see that it's possible to test voting schema/store function independently in `vote_test.go`, but does it make sense for both the events test and the actual application to ensure voting store is ready via the event store constructor?  That's the main thing I figured we'd gain from this.